tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -143,6 +145,139 @@ def test_fail_on_multiple_drivers():
         extract_db_type_from_uri("mysql+pymsql+pyodbc://...")
 
 
+@dataclass
+class TestDatabase:
+    db_url: str
+    store: SqlAlchemyStore
+
+
+@pytest.fixture
+def test_database() -> TestDatabase:
```

Comment:
```suggestion
def test_database(tmp_path) -> TestDatabase:
```

We can use the `tmp_path` fixture.